MOBILE_API.xml

```diff
@@ -449,6 +449,7 @@
         <element name="APP_UNAUTHORIZED" since="2.0" />
         <element name="PROTOCOL_VIOLATION" since="4.0" />
         <element name="UNSUPPORTED_HMI_RESOURCE" since="4.1" />
+        <element name="RESOURCE_CONSTRAINT" since="6.1" />
```

Comment:
@mvorobio Since 6.2, isn't it?

Reply:
@AByzhynar please check https://github.com/mvorobio/rpc_spec/pull/1/commits/8aa677c4a02da1c18908f5a695fde34859baacd1